support.py

```diff
@@ -0,0 +1,67 @@
+import sublime
+import sublime_plugin
+
+from .cron_descriptor import cron_descriptor
```

Comment:
I'd suggest having the import happen as lazily as possible. I.e. in the try-except block.

Additionally, it might not be the worst idea to have Import and Module Error's caught there and reported which might make remote debugging a bit easier?